tests/test_geom.py

```diff
@@ -933,8 +935,11 @@ def test_deprecation_warnings():
 
 
 def test_filter():
-    _keep = lambda x, y: True
-    _drop = lambda x, y: False
+    def _keep(x, y):
+        return True
+
+    def _drop(x, y):
+        return False
```

Comment:
Same here - I didn't intentionally make this change although I did Black format.